ippon/test_registration_views.py

```diff
@@ -31,12 +40,22 @@ def test_register_unique_user_with_valid_payload_creates_new_user(self, email_us
             message="You have been successfully registered in ippon with username {}".format(
                 self.valid_payload["username"]))
 
-    def test_register_with_not_unique_username_does_not_create_new_user(self):
+    def test_non_unique_username_response(self):
```

Comment:
I wonder if it wouldn't be better to leave two tests for that.
1. It would let those tests get more descriptive name
2. Those are two different mechanisms IMHO - error reporting for machines and error reporting for puny humans.
... or just change the test name to something like test_non_unique_user_name_registration_results_in_400_and_unique_name_error_content
this is a hobby project, we don't have daily keystrokes limit here :P
Besides that, the test name is not consistent with the first test name in this file and I think all other test names in project.